2020F-usfca-btec-640.github.io/.gitignore

```diff
@@ -0,0 +1,5 @@
+_site
```

Comment:
This file should not be a part of the PR. Please remove it so we don't have to deal with merge conflicts.